jsonify.js

```diff
@@ -27,8 +26,7 @@ var lf_processArguments = function(){
 
     while (0 < l_aryArgs.length){
         l_strFlag = l_aryArgs.shift();
```

Comment:
I tend to like it when the variables names have a stronger association to their ancestors. So, for example, consider an array of arguments and an individual argument. A strong correlation between the two would be `l_aryArgs` and `l_strArg`.

In this case, however, I think mixing flags with args might cause confusion to someone reading the script the first time. Would possible improvement might be 'l_aryArgFlags` and `l_strArgFlag`.